core/unix/coredump.c

```diff
@@ -484,11 +485,23 @@ os_dump_core_internal(dcontext_t *dcontext)
     // of LOAD program header (number of sections minus one since shstrtab does not
     // require a LOAD header).
     const ELF_OFF program_header_count = section_count;
+    // core_file_offset stores the location (offset) of the content in the core
```

Comment:
"content" is not clear to me: this is the current offset to the next section data to write?

Reply:
Yes, it's the current offset to the next section data. I've updated the comment accordingly.